pages/api/v1/leaderboard/user-points.ts

```
@@ -0,0 +1,74 @@
+			status: Number(error.name) || 500
+		};
+	}
+};
+
+const handler: NextApiHandler<LeaderboardPointsResponse | MessageType> = async (req, res) => {
+	storeApiKeyUsage(req);
+
+	//get user_id from req.query
+	const { page = 1, user_id } = req.query;
+
+	if (isNaN(Number(page)) || isNaN(Number(user_id))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)